tlspuffin/harness/wolfssl/src/put.c

```diff
@@ -26,7 +26,7 @@
 #define TYPETIME word32
 #endif
 
-//#define TIME_CHANGE
+// #define TIME_CHANGE
```

Comment:
useful?

Reply:
I think that automatic formatting that changed that, maybe it's not that problematic ?

Reply:
No, it is not :)